log/test.log

```diff
@@ -0,0 +1,55 @@
+{"app":{"@timestamp":"2020-01-07T10:54:47.429-08:00","severity":"DEBUG","host":"nblhedberg.local","pid":20017,"tid":"oufqr6x40","service_name":null,"logfile":"/Users/lhedberg/invoca/elasticsearch_models/log/test.log","fiber":null,"message":"Adapter: considering polling ...","timestamp":"2020-01-07T10:54:47.429-08:00","progname":null}}
```

Comment:
Looks like we should add `/log/` to the `.gitignore` so this won't be included

Reply:
Oops, sorry I must have missed this earlier. It's been addressed now.